webpack.config.js

```diff
@@ -15,9 +15,22 @@ module.exports = {
         path: outPath,
         filename: 'bundle.js',
     },
-    performance: {
-        maxEntrypointSize: 716800,
-        maxAssetSize: 716800
+    optimization: {
+        namedChunks: true,
```

Comment:
Why do we need to split?
Does it make a difference?

Reply:
I thought the whole browser page may appear quicker on slow network connections.
But I haven't really test this properly. Need to investigate more

Reply:
I don't it'll make any difference, it's very small. But doesn't matter

Reply:
Ok. I actually do not like to have it split into chunks. So I am going to remove it later if you also think it does not make any differences

Thank you